src/viur/core/bones/base.py

```diff
@@ -187,6 +187,12 @@ class BaseBone(object):
     type = "hidden"
     isClonedInstance = False
 
+    _owner = None
```

Comment:
```suggestion
    _skel_cls = None
```
Can you rename this into `skel_cls`?

Reply:
Done in afca4d1b.

I've also removed the underscore, as in the methods it's ok to access it from outside and there's no name collision.